content/Concepts/commands.mdx

```diff
@@ -1,57 +1,181 @@
+  - If the value is an empty string, the Config will be deleted from the ConfigStore.
 
-## Select How To Run (Locally, Or On The CI Server)
-
-If you intend to run interactively, you'll need to make sure you're logged in and your CLI is 
-connected and set to the correct organization. If you're setting up Configu to run on the CI/CD, 
-[you'll need a token](/administration) as the tool suggests.
-
-## Select How You'd Like The Config Exported
-
-Configu can deliver your configuration in a variety of ways:
-
- - **Print**: Print the configuration to standard output, or to a file, and select the desired format
- - **Template**: Output the configuration to a template file
- - **Source**: Export the configuration in a way that shell scripts/docker files can evaluate it
- - **Sync**: Synchronize the configuration directly to a supported runtime / IAAS / PAAS provider
-(like Heroku, AWS, and more)
-
-Using the command generator will also help to understand which flags Configu expects to see with each 
-choice.
-
-## Copy And Run The Command!
-
-You'll see the command preview as you build it in the app. Once you're done, you can copy it, paste, 
-insert any tokens that are required and Configu will handle the rest - here's what will happen:
-
- 1.  Configu will make sure your client has the permissions to execute the command
- 2.  Configu will look up the specified schema and set, stored in the selected stores
- 3.  Configu will expand any template strings, and fetch any required secrets from the specified secret managers
- 4.  Configu will complete the operation by outputting the config according to the flags you specified
-
-## Don't Forget - Our Client Libraries Are Open Source!
-
-We made the client libraries open source! We want you to add your own formats and runtimes if they 
-don't already exist.
\ No newline at end of file
+#### Usage
+
+```bash
+configu upsert \
+  --store "{getStoreConnectionStringPlaceholder('configu')}" \
+  --set "production" --schema "./service.cfgu.json" \
+  --config "API_TOKEN=123abc" --config "AWS_REGION=us-east-1" --config "DELETE_ME="
+```
+
+```js
+import path from 'path';
+import { ConfiguStore, ConfigSet, ConfigSchema, UpsertCommand } from '@configu/node';
+
+(async () => {
+  try {
+    if (!process.env.CONFIGU_ORG || !process.env.CONFIGU_TOKEN) {
+      return;
+    }
+
+    const store = new ConfiguStore({
+      credentials: {
+        org: process.env.CONFIGU_ORG,
+        token: process.env.CONFIGU_TOKEN,
+        type: 'Token',
+      },
+    });
+    const set = new ConfigSet('production');
+    const schema = new ConfigSchema(path.join(__dirname, 'service.cfgu.json'));
+
+    await new UpsertCommand({
+      store,
+      set,
+      schema,
+      configs: [
+        { key: 'API_TOKEN', value: '123abc' },
+        { key: 'AWS_REGION', value: 'us-east-1' },
+        { key: 'DELETE_ME', value: '' },
+      ],
+    }).run();
+  } catch (error) {
+    console.error(error);
+  }
+})();
+```
```

Comment:
Please end with a description  that explains what the snippets show (adding/updating 2 configs and deleting the third)